src/core/internal/backtrace/handler.d

```diff
@@ -0,0 +1,116 @@
+/**
+ * Libunwind-based implementation of `TraceInfo`
+ *
+ * This module exposes an handler that uses libunwind to print stack traces.
+ * It is used when druntime is packaged with `DRuntime_Use_Libunwind` or when
+ * the user uses the following in `main`:
+ * ---
+ * import core.runtime;
+ * import core.internal.backtrace.handler;
+ * Runtime.traceHandler = &libunwindDefaultTraceHandler;
+ * ---
+ *
+ * Authors: Mathias 'Geod24' Lang
+ * Copyright: D Language Foundation - 2020
+ * See_Also: https://www.nongnu.org/libunwind/man/libunwind(3).html
+ */
+module core.internal.backtrace.handler;
+
+import core.internal.backtrace.dwarf;
+import core.internal.backtrace.libunwind;
+import core.internal.backtrace.primitives;
+
+/// Ditto
+public class LibunwindHandler : TraceInfo
```

Comment:
Do you want all this to be public in the `internal` package?

Reply:
Ah, I have a habit of putting protection qualifier before the symbol, even when `public`.
Originally yes, but then I thought of adding `libunwindDefaultTraceHandler`, so this could be `package(core)`.